static/src/js/containers/AuthCallbackContainer/AuthCallbackContainer.js

```diff
@@ -39,42 +42,40 @@ export const AuthCallbackContainer = ({
       redirect = '/'
     } = params
 
-    // Verify that the redirect params are real URLs
-    try {
-      let redirectUrl
-      if (eddRedirect) redirectUrl = new URL(eddRedirect)
-      if (redirect && redirect !== '/') redirectUrl = new URL(redirect)
-
-      if (
-        redirectUrl
-        && redirectUrl.protocol !== 'http:'
```

Comment:
Seems like we lost where were checking for the allowed protocols. Is that intentional?

Reply:
Yea that is intentional. It doesn't sufficiently cover the strictness we need to properly handle url redirects. Here we are now only allowing redirects to addresses relative to EDSC or earthdata-download

Reply:
For instance one value I see in testing with downloading a collection where EDL auth was required is `earthdata-download://authCallback?fileId=3&ee=prod&token=<my-token>`
I'm not sure we know what the fileId is going to be or really that the token matches whatever token we have saved